perception/lidar_apollo_segmentation_tvm_nodes/CMakeLists.txt

```diff
@@ -19,7 +19,8 @@ find_package(autoware_cmake REQUIRED)
 autoware_package()
 
 # Check that a neural network was available for lidar_apollo_segmentation_tvm to be built.
-if(lidar_apollo_segmentation_tvm_FOUND)
+autoware_check_neural_network(baidu_cnn "${${lidar_apollo_segmentation_tvm}_BACKEND}")
```

Comment:
@mitsudome-r Is there something wrong with checking if the "core" package has been found when building this one? Because that's fundamentally what we want to check here.
With this change `lidar_apollo_segmentation_tvm` is not a variable, so this package will probably not get built even when the correct cmake variable is set by the user.